.github/workflows/workflow.yml

```diff
@@ -0,0 +1,31 @@
+name: NIST COBOL85 tests
+on: [push]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout opensource COBOL
+      - name: Checkout opensource COBOL
+        uses: actions/checkout@v2
+        
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y build-essential libncurses5 libncurses5-dev libgmp-dev bison flex gettext automake autoconf
```

Comment:
That should be

```diff
-sudo apt-get install -y build-essential libncurses5 libncurses5-dev libgmp-dev bison flex gettext automake autoconf
+sudo apt-get install -y build-essential libncurses-dev libgmp-dev bison flex gettext automake autoconf
```

Reply:
This correction is reflected in the following commit
https://github.com/opensourcecobol/opensource-cobol/commit/93def13076e056d9a31be701b0ae080fce02f6d7